src/components/Main/stories.tsx

```diff
@@ -1,19 +1,19 @@
 import { Story, Meta } from '@storybook/react/types-6-0'
-import { withKnobs, text } from '@storybook/addon-knobs'
 import Main from '.'
 
 export default {
   title: 'Main',
   component: Main,
-  decorators: [withKnobs]
+  args: {
+    title: 'title default',
+    description: 'description default'
+  }
 } as Meta
 
-export const Basic: Story = () => (
-  <Main
-    title={text('Title', 'React Avançado')}
-    description={text(
-      'Description',
-      'TypeScript, ReactJS, NextJS e Styled Components'
-    )}
-  />
-)
+export const Basic: Story = (args) => <Main {...args} />
+Basic.args = {
+  title: 'title basic',
+  description: 'description basic'
+}
+
+export const Default: Story = (args) => <Main {...args} />
```

Comment:
New story just to show that we can have default values coming from the export default and we can overwrite directly via the story itself, like it's done on line 14 with `Basic.args`